tls_manager.go

```diff
@@ -156,7 +142,16 @@ func (t *TLSManager) getConfig() ([]grpc.ServerOption, []grpc.DialOption,
 	// We set this to 200MiB atm. Should be the same value as maxMsgRecvSize
 	// in cmd/lncli/main.go.
 	restDialOpts := []grpc.DialOption{
```

Comment:
The comment above should be changed to indicate that we skip tls for REST calls.